src/db/services/endpoints/index.js

```diff
@@ -8,22 +10,25 @@ exports.createEndpoint = body => {
 }
 
 exports.readEndpoint = endpointId => {
-  return EndpointModel.findById(endpointId)
+  const objectId = new ObjectId(endpointId)
+  return EndpointModel.findById(objectId)
```

Comment:
If the endpointId is invalid an error will be thrown here - `Error - Argument passed in must be a single String of 12 bytes or a string of 24 hex characters`. This method is invoked in a try and catch and the error above is what will be sent back to the user. The error message is not very helpful. The message that was being sent back, before this change, for invalid endpointIds is actually a bit better